openmetadata-ui/src/main/resources/ui/src/components/SampleDataTopic/SampleDataTopic.tsx

```diff
@@ -13,8 +13,10 @@
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Typography } from 'antd';
+import { t } from 'i18next';
```

Comment:
@ShaileshParmar11 , Can you check if we are using this, I can see you have imported `useTranslation` as well.

Reply:
I will have a look thanks!